Fedora support for CHIP-tools

hadess opened the pull request:
The first 3 commits should be low-risk, and certainly allowed me to flash my CHIP with Debian.

Not sure how to handle the third change. I think that the script should work equally under Python2 and Python3, but I'd be happy to make it detect which version of Python is available, and launch the right interpreter.

soderstrom-rikard commented:
I think the checks for a properly installed fel is really good (sanity checks always are).

Renaming verify.sh to verify.py should probably be more of a maintainers choice (aka NextThingCo). 
Just like you noticed yourself, your changes to the verify script is python 3 only.

My suggestion, if I may be so bold ^^. Is to make this patch set, sanity checks only. Then when both the #3 and this one could be merged and it should work on all systems.

Do you think this would be a good idea?

hadess commented:
The maintainer can choose which commits to cherry-pick, I'll leave it as-is for now.

johan-bjareholt commented:
I started rewriting parts of the CHIP-tools myself because i (just like you) found that fel has been renamed sunxi-fel and that the python script is a mess.

This have been here for half a year soon, how can this possibly not have been merged yet? These are incredibly simple patches.

hadess commented:
I rebased again, and added a bug fix that checks for the presence of fel (took me 20 minutes to figure out that the flashing didn't work because of a missing package instead of a broken cable).

aruiz commented:
What's the status of this PR?
